### ISSUE TITLE
Offline map examples 

### ISSUE THREAD

**Issue by spyhunter99** [NASAWorldWind/WorldWindAndroid#3]
Is offline map data on the roadmap?

Also, if there's any examples on how to incorporate a TMS or Slippy based map source (web mercator), that would be awesome.

**Comment by pdavidc** [comment]
Yes, In the short term we are planning on incorporating offline map data in the form of a local file cache, which the most recently viewed data from a WMS. In the long term, we're looking at GeoPackage as a possible source of application specified offline map data.

**Comment by lmorda** [comment]
Hi @pdavidc is this still on the roadmap?  My PM is asking for a timeline on a map project I'm working on, and offline maps is a requirement.  Thanks!

**Comment by pdavidc** [comment]
Yes, offline maps for remote resources (WMS, WMTS, WCS, image URL, etc.) is absolutely on our roadmap. We don't have a timeline for this capability, but it's on our list of what to work on next.

Note that WWAndroid currently has an offline map capability in GeoPackage. This capability is currently limited to GeoPackage image pyramids in the EPSG 4326 projection. See the GeoPackage tutorial for a usage example.

**Comment by EMaksymenko** [comment]
Hi. I have a goal to implement offline maps cache in community edition fork this month.
I was planed to use MBTiles database format to store cache and avoid lack of inodes on file system.

Does anybody from NASA development team have a time to make a conference call with me to discuss initial view of file cache implementation? I want to relay on your initial view to be able to pull request this feature later in the master branch.

**Comment by CodeK1988** [comment]
@ComBatVision hi mate, did you successfully load mbtiles files?
